terraform/aws/modules/compute/main.tf

```diff
@@ -104,29 +104,42 @@ resource "aws_ecs_service" "km_ecs_service" {
     container_port   = 80
   }
   network_configuration {
-    assign_public_ip = true
+    assign_public_ip = false
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure public IP address is not assigned to AWS ECS service**
 
**Vulnerability**
AWS ECS services is configured to automatically assign public IP addresses. This makes your ECS instances accessible from internet.
 
**Remediation**
In AWS Console - 
 In AWS Console - 
 1. Sign in to the AWS Console and go to the Amazon ECS console.
 3. Select Networking.
 4. Select Auto Assign Public IP to DISABLED.
 In Terraform. Edit 'aws_ecs_service' resource and set attribute 'assign_public_ip' to false.

source: tenablecs-